get-started/index.md

```diff
@@ -234,21 +242,17 @@ web client, continue on to [Guides]({% link guides/index.md %}) for
 guides through various common actions and workflows.
 
 ## [Uninstalling Cryostat Operator](#uninstalling-cryostat-operator)
-In order to ensure that objects created by the operator and recordings created
+In order to ensure that objects created by the Operator and recordings created
 by Cryostat are properly removed, the Cryostat Operator must remain installed
 when attempting to delete the Cryostat custom resource, or any Recording
 custom resources.
 
 To completely remove Cryostat and all objects and recordings created by it:
-1. Delete any Recording custom resources.
-    - If Cryostat is no longer deployed, you must redeploy it by creating a
-      Cryostat custom resource.
-    - If the Cryostat Operator has already been uninstalled, please reinstall it
-      before deleting any Recording custom resources.
-2. Delete the Cryostat custom resource.
+1. Delete the `Cryostat` or `ClusterCryostat` custom resources (`CR`s) and
```

Comment:
How about following this guide: https://docs.openshift.com/container-platform/4.12/operators/admin/olm-deleting-operators-from-cluster.html?

We probably just need to either use the UI or delete the `Subscription` object.